samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -198,15 +238,44 @@ static void wifi_send_raw_tx_packets(void)
 
 	buf_length = sizeof(struct raw_tx_pkt_header) + sizeof(test_beacon_frame);
 	memcpy(test_frame, &packet, sizeof(struct raw_tx_pkt_header));
-	memcpy(test_frame + sizeof(struct raw_tx_pkt_header),
+
+	if (num_pkts == 1) {
+		memcpy(test_frame + sizeof(struct raw_tx_pkt_header),
 				&test_beacon_frame, sizeof(test_beacon_frame));
 
-	ret = wifi_send_raw_tx_pkt(sockfd, test_frame, buf_length, &sa);
-	if (ret < 0) {
-		LOG_ERR("Unable to send beacon frame: %s", strerror(errno));
-		close(sockfd);
-		free(test_frame);
-		return;
+		ret = wifi_send_raw_tx_pkt(sockfd, test_frame, buf_length, &sa);
+		if (ret < 0) {
+			LOG_ERR("Unable to send beacon frame: %s", strerror(errno));
+			close(sockfd);
+			free(test_frame);
+			return;
+		}
+	} else {
+		while (1) {
+			memcpy(test_frame + sizeof(struct raw_tx_pkt_header),
+					&test_beacon_frame, sizeof(test_beacon_frame));
+
+			if (transmission_mode == FIXED_MODE_TRANSMISSION && num_pkts == 0) {
+				break;
+			}
+
+			ret = sendto(sockfd, test_frame, buf_length, 0,
+					(struct sockaddr *)&sa, sizeof(sa));
+			if (ret < 0) {
+				LOG_ERR("Unable to send beacon frame: %s", strerror(errno));
+				close(sockfd);
+				free(test_frame);
```

Comment:
Shouldn't we continue sending even if a packet fails? We are bound to see some intermittent failures.
And we should add a summary once the transmission is done for non-continuous packets:

`Sending num_pkts  with num_pkts_fails`  or something in that lines.

Reply:
As in the above comment, it is decided to send MAX_UINT packets, adding ``Sending num_pkts  with num_pkts_fails`` print in both the cases.